internal/storage/storage.go

```diff
@@ -65,3 +70,45 @@ func GetObjectsPage[T Paginated](s Storage, address string, from, count uint64)
 	})
 	return
 }
+
+func GetAccountsPage[T Paginated](s Storage, from, count uint64) (ret []T, pagination *models.PaginatedResponse) {
+
+	pagination = new(models.PaginatedResponse)
+	pagination.Start = from
+	accountMap := s.GetAccounts()
+	storageAccountMap := s.HolderModelsToStorage(accountMap)
```

Comment:
Better to convert it only for elements that you are returning in the cycle below